### PR TITLE
job-builder: fix ci/cd steps

### PR DESCRIPTION
**Pull request by mivanov1988**
# Why
The job builder steps failed due to the docker version - https://gitlab.com/vmware-analytics/versatile-data-kit/-/jobs/3666206273 .
# What
Revert the docker version for the job builder steps.
# Testing done
CI/CD

### CODE REVIEW

**Review by dakodakov — APPROVED** [review]

File: projects/control-service/cicd/.gitlab-ci.yml [hunk 1]
```diff
@@ -121,7 +121,12 @@ control_service_test_helm_chart:
       - projects/control-service/projects/helm_charts/pipelines-control-service/version.txt
 
 control_service_publish_job_base_image:
-  extends: .images:dind
+  image: docker:19.03.15
```
Comment (ivakoleva) [inline projects/control-service/cicd/.gitlab-ci.yml:124]: 20.10.23 (in favour of [20.10.22](https://github.com/vmware/versatile-data-kit/blob/main/projects/control-service/cicd/.gitlab-ci.yml#L8)) does not solve the issue? what was the issue?

File: projects/control-service/projects/job-builder-secure/version.txt [hunk 2]
```diff
@@ -1 +1 @@
-1.0.0
+1.1.0
```
Comment (ivakoleva) [inline projects/control-service/projects/job-builder-secure/version.txt:1]: nit: do we still use the version.txt mechanism or is it redundant?

**Review by ivakoleva — APPROVED** [review]

**Comment by tozka** [comment]
Why do we need older version? What is the difference between the new and the old?

File: projects/control-service/projects/job-builder-secure/version.txt (hunk 2 above)
Comment (mivanov1988) [reply projects/control-service/projects/job-builder-secure/version.txt:1]: we use it for job-builder

**Comment by mivanov1988** [comment]
> Why do we need older version? What is the difference between the new and the old?

Just reverted the version in order to unblock the pipeline - more details here - https://gitlab.com/vmware-analytics/versatile-data-kit/-/jobs/3666206273.

File: projects/control-service/cicd/.gitlab-ci.yml (hunk 1 above)
Comment (mivanov1988) [reply projects/control-service/cicd/.gitlab-ci.yml:124]: Just reverted the version in order to unblock the pipeline - more details here - https://gitlab.com/vmware-analytics/versatile-data-kit/-/jobs/3666206273.

**Comment by mivanov1988** [comment]
> Why do we need older version? What is the difference between the new and the old?

I don't know what is the difference between the new and the old - @murphp15 do you know?

**Pull request merged**